src/targets/gpu/target.cpp

```diff
@@ -103,6 +103,10 @@ std::vector<pass> target::get_passes(migraphx::context& gctx, const compile_opti
         unsupported_fp8_ops.insert("dot");
         unsupported_fp8_ops.insert("quant_dot");
     }
+#else
+    // mlir doesn't support fp8 dot
+    unsupported_fp8_ops.insert("dot");
+    unsupported_fp8_ops.insert("quant_dot");
```

Comment:
This shouldn't be necessary. 
https://github.com/ROCm/AMDMIGraphX/blob/7011bda8a316bdff7b43a873ff0d6088155bcf0a/src/targets/gpu/fuse_mlir.cpp#L234 

MIGraphX doesn't pick rocMLIR for the FP8 dot/quant_dots. 

Do you have a case where rocBLAS is disbaled in MIGraphX ?

Reply:
Yes, that is required when rocBLAS is not used (see `MIGRAPHX_USE_ROCBLAS=OFF`).

Reply:
Yes, this is added for case where rocBLAS is disabled in MIGraphX. There is variable MIGRAPHX_USE_ROCBLAS in develop branch, so when it is set to OFF we need to add unsupported fp8 ops for dot and quant dot.